src/components/Footer/index.tsx

```diff
@@ -1,44 +1,40 @@
-import React, {
-  useEffect
-} from 'react';
+import React, { useEffect } from 'react';
 
 import {
-  Button,
-  Divider
+  Button, Divider
 } from 'antd';
 
 import OlControlMousePosition from 'ol/control/MousePosition';
 import OlControlScaleLine from 'ol/control/ScaleLine';
-import {
-  createStringXY
-} from 'ol/coordinate';
+import { createStringXY } from 'ol/coordinate';
 
-import {
-  useTranslation
-} from 'react-i18next';
+import { useTranslation } from 'react-i18next';
 
 import ScaleCombo from '@terrestris/react-geo/dist/Field/ScaleCombo/ScaleCombo';
 import useMap from '@terrestris/react-geo/dist/Hook/useMap';
 
 import './index.less';
+import useAppSelector from '../../hooks/useAppSelector';
+import { Legal } from '../../store/legal';
```

Comment:
This looks like it could have linting issues, please check,

Reply:
Checked it locally and all checks here run successfully. `.eslintrc.js` says curly new line warning starts at two properties. VS Code edited the entries with just one property automatically on save